doc-Installing_on_AWS/topics/installation.adoc

```diff
@@ -181,17 +181,21 @@ To import the appliance:
 ]
 ----
 +
-. Use AWS-CLI tools to import the image. Run the following command:
+. Use AWS-CLI tools to import the disk as a snapshot by following the steps in the
+link://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-import-snapshot.html[Importing a Disk as a Snapshot Using VM Import/Export] documentation.
 +
 ----
-$ aws ec2 import-image --disk-containers file://containers.json --region BUCKET_REGION
+$ aws ec2 import-snapshot
 ----
 +
-. Check the progress of your imported image by running the following command:
+. Create the AMI from the snapshot by following the steps in the
+link://docs.aws.amazon.com/AWSEC2/latest/UserGuide/creating-an-ami-ebs.html[Creating a Linux AMI from a Snapshot] documentation.
 +
 ----
  $ aws ec2 describe-import-image-tasks --region BUCKET_REGION --import-task-ids import-ami-AMI_ID_GOT_FROM_RESPONSE
 ----
 +
-
-
+. Check the progress of your imported image by running the following command:
+----
+ $ aws ec2 describe-import-image-tasks --region BUCKET_REGION --import-task-ids import-ami-AMI_ID_GOT_FROM_RESPONSE
+----
```

Comment:
Checking the progress should happen after running `import-snapshot` command (but before creating AMI from snapshot) and the command should be `aws ec2 describe-import-snapshot-tasks --import-task-ids import-ami-AMI_ID_GOT_FROM_RESPONSE`

Also, it should probably say check progress of snapshot, not image.

Reply:
Moved this step up, and included  entire command line shown.

Reply:
Changed command line with reference to AWS documentation for options to use with $ aws ec2 register-image